plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui.test/.classpath

```diff
@@ -2,6 +2,10 @@
 <classpath>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src/"/>
+	<classpathentry kind="src" path="src/">
+		<attributes>
+			<attribute name="test" value="true"/>
```

Comment:
Need to revert this. The attribute "test=true" doesn't seem right against "src/".

Reply:
I wonder why this keeps getting added? And whether .classpath files belong in the repo in the first place?